### DISCUSSION TITLE
Accent color in dark theme?

### DISCUSSION THREAD

**Discussion by thetrotfreak** [flet-dev/flet#1133] [General]
Are there any resources where I can read about why exactly `color_scheme_seed` doesn't work when `page.theme_mode` is set to `ThemeMode.DARK`?

**Comment by ndonkoHenri** [comment]
Make use of the [`page.dark_theme`](https://flet.dev/docs/controls/page#dark_theme) also. `page.theme` is for the light mode.

**Reply by thetrotfreak** [reply]
@ndonkoHenri, OMG, a huge thanks to you! It was so disappointing to see that BLUE accent on dark theme. Finally, I get to rest in ease.
